recipe/meta.yaml

```diff
@@ -1,22 +1,35 @@
 {% set name = "igraph" %}
-{% set version = "0.8.3" %}
+{% set version = "0.9.0" %}
 
 package:
   name: {{ name }}
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.tar.gz  # [not win]
-  url: https://github.com/igraph/igraph/releases/download/{{ version }}/{{ name }}-{{ version }}.tar.gz  # [not win]
-  sha256: cc935826d3725a9d95f1b0cc46e3c08c554b29cdd6943f0286d965898120b3f1  # [not win]
-
-  fn: {{ name }}-{{ version }}.zip  # [win]
-  url: https://github.com/igraph/igraph/releases/download/{{ version }}/{{ name }}-{{ version }}-msvc.zip  # [win]
-  sha256: a6ab981c88b31bffb833a077235f1e3e46d4576a254c9a03397dcc7ebf90e942  # [win]
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/igraph/igraph/releases/download/{{ version }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 012e5d5a50420420588c33ec114c6b3000ccde544db3f25c282c1931c462ad7a
+  patches:
+    - correct-test-path.patch
+    - 0001-feat-allow-external-f2c-arith.h-headers-when-cross-c.patch
+    - 0002-tests-CMake-added-pre-existing-linker-flags.patch
 
 build:
   skip: True  # [win and vc<14]
-  number: 2
+  number: 0
+  script_env:
+   - ENABLE_LTO=1
+   - ENABLE_LTO=0  # [osx]
```

Comment:
why not enable lto for osx?

Reply:
I got an error on CLang 11.0.1: `Link-time optimization not supported on this compiler`. If it is possible to enable it in some way, that would be great!

Reply:
Perhaps it had something to do with the missing `${CMAKE_ARGS}`, I could try to enable it again.

Reply:
Yes, that was probably the issue.